### DISCUSSION TITLE
如果要學習網頁，你不可不學...php

### DISCUSSION THREAD

**Discussion by dabbie109303501** [1101-GS4719-C/Python-Community-GS4719-C#905] [General]
php的簡介與重要性(但好像越來越少人學php了...)
https://kamadiam.com/what-is-php/

php語法支援
https://www.w3schools.com/php/default.asp

個人經驗補充:
如果要在html的頁面之間傳值(ex.登入頁面，要將使用者輸入的帳密傳到下一個網頁上)的話，php是個好用的工具
而且php可以透過echo的方式包javascript跟html
ex.
你想要寫的(div的格式都一樣但內容不一樣)
```
<html>
 .  .  .
    <div style="...">
    123
    </div>
    <div style="...">
    456
    </div>
    <div style="...">
    789
    </div>
 .  .  .
</html>
```

那你可以用php這樣寫:
```
<?php
$num=array("123","456","789");
?>
<html>
 .  .  .
    <?php
        for($x=0;$x<3;$x++){
             echo "<div style="...">".$num[$x]."</div>";
        }
    ?>
 .  .  .
</html>

```
像是這樣:
![image](https://user-images.githubusercontent.com/75252877/207530704-efdbebd0-abed-48d1-9595-df30e0cd70d1.png)

我不知道這樣寫是不是最佳解，但當時發現可以這樣寫的時候我覺得很快樂 😍

**Comment by evanchou1204** [comment]
感覺現在真的比較少人學php了，在看這篇文之前我也沒有聽過，但真的沒有最爛的程式語言，只是在不同情境下哪個比較適合而已。

**Comment by Doreen331** [comment]
之前甚至有看過別人分享過時的語言，勸人不要再學，其中有一個就是php，但我認為多學是無妨，至少可以和樓主一樣發現一個讓人快樂的寫法😍

**Comment by Evan0979** [comment]
程式渣渣的我看不大懂,但還是感謝分享!!
